yarn.lock

```diff
@@ -64,7 +64,37 @@
     semver "^6.3.0"
     source-map "^0.5.0"
 
-"@babel/generator@^7.15.0", "@babel/generator@^7.7.2":
+"@babel/core@^7.15.0":
```

Comment:
As we can see here, the new babel core version was added, but the old one is still here.